pegasus/sites.v3/code.org/public/maker/index.haml

```diff
@@ -3,118 +3,101 @@ title: Become a Maker | Learn about Physical Computing
 theme: responsive_full_width
 ---
 
-%script{data: {"amplitude-api-key": CDO.safe_amplitude_api_key}}
-%script{data: {'statsig-api-client-key': CDO.safe_statsig_api_client_key}}
-%script{data: {'managed-test-server': "#{CDO.running_web_application? && CDO.test_system?}"}}
 %link{href: "/css/generated/design-system-pegasus.css", rel: "stylesheet"}
-%link{href: "/css/generated/maker-styles.css", rel: "stylesheet"}
+%link{href: "/css/generated/page/maker-styles.css", rel: "stylesheet"}
 
 -# FontAwesome icons
 - icon_microchip = "fa-solid fa-microchip"
 - icon_hands_clapping = "fa-solid fa-hands-clapping"
 - icon_face_smile = "fa-solid fa-face-smile"
 
-%section.hero-banner-basic{style: "background: #00736E"}
-  .wrapper
-    %div.flex-container
-      %div.text-wrapper.col-50
-        %h1.white Become a Maker
-        %p.white Explore your creativity with physical computing.
-      %div.col-30.flex-container.justify-end
-        %img{src: "/images/maker/maker-skinny-banner-img-left.svg", alt: ""}
-        %img{src: "/images/maker/maker-skinny-banner-img-right.svg", alt: ""}
-    .clear
+%section.hero-banner-basic.overlay
+  .wrapper.grid-container.justify-space-between.align-items-center.gap-4.wider-text
+    .text-wrapper
+      %h1.white
+        =hoc_s("maker_page.top.heading")
+      %p.body-one.no-margin-bottom.white
+        =hoc_s("maker_page.top.desc")
+    %figure
+      %img{src: "/images/maker/maker-banner-image.svg", alt: "", style: "width: 100%"}
 
 %section
   .wrapper
-    .text-wrapper.col-60
-      %h2 What is a Maker?
-      %p <strong>Makers invent, explore, innovate, and create using technology like development boards, robotics, 3D printing, programming, and more.</strong>
-      %p.body-three Maker education takes STEM (Science, Technology, Engineering, Mathematics) a step further by inviting hands-on experience and collaboration into the classroom to help teach problem solving and critical thinking.
-    %div.col-33{style: "float: right"}
-      %img{src: "/images/maker/maker-students-cp.png", alt: "", style: "width: 100%; margin-top: .75em"}
-    .clear
+    .grid-container.two-col.justify-space-between.align-items-center.gap-4
+      %figure
+        %img.rounded-corners{src: "/images/maker/maker-students-cp.png", alt: "", style: "width: 100%"}
+      .text-wrapper
+        %h2=hoc_s("maker_page.what.heading")
+        %p.heading-xs{style: "margin-bottom: 1rem"}
+          =hoc_s("maker_page.what.desc_01")
+        %p=hoc_s("maker_page.what.desc_02")
 
+-# TODO - remove the divider line and unhide case study section when we get the link
```

Comment:
This is how this section will look once we get the url and remove the comments.
![localhost code org_3000_maker(Code org - 1366x768)](https://github.com/code-dot-org/code-dot-org/assets/9256643/1a34f972-6bc9-4452-9e47-59fc1f78140b)